interface/HHAnalyzer.h

```diff
@@ -105,20 +109,21 @@ class HHAnalyzer: public Framework::Analyzer {
         void fillTriggerEfficiencies(const Lepton & lep1, const Lepton & lep2, Dilepton & dilep);
 
         // global event stuff (selected objects multiplicity)
-        BRANCH(nJets, unsigned int);
+        ONLY_NOMINAL_BRANCH(nJets, unsigned int);
         BRANCH(nJetsL, unsigned int);
```

Comment:
If I remove these we have no longer a way to know the number of leptons passing the selection other than looping over the `hh_leptons` collection and counting.